code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -178,11 +179,19 @@
 	SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "chemical_laughter", /datum/mood_event/chemical_laughter)
 	..()
 
+/datum/reagent/consumable/laughter/overdose_process(mob/living/carbon/M, delta_time, times_fired)
+	if(DT_PROB(10, delta_time))
+		to_chat(M, span_userdanger("You laugh so hard you lose your breath..."))
+		M.adjustOxyLoss(5)
+		M.emote("gasp")
+	..()
```

Comment:
```suggestion
	return ..()
```

Reply:
Is there a particular reason for this?  Most reagent code in this file uses:

```
..()
```

or

```
..()
. = TRUE
```

Why would we want to return the parent object?